login.spec.js

```diff
@@ -0,0 +1,47 @@
+describe('Login', () => {
+  beforeEach(() => {
+    cy.visit('/login');
```

Comment:
The 'beforeEach' block is visiting the login page before every test. If the tests are independent and do not require a fresh state of the login page, consider moving this to a 'beforeAll' block to reduce the number of navigations.